frontend/src/metabase/lib/dom.js

```diff
@@ -495,23 +495,10 @@ export function initializeIframeResizer(onReady = () => {}) {
       onReady,
     };
 
-    // FIXME: Crimes
-    // This is needed so the FE test framework which runs in node
-    // without the avaliability of require.ensure skips over this part
-    // which is for external purposes only.
-    //
-    // Ideally that should happen in the test config, but it doesn't
-    // seem to want to play nice when messing with require
-    if (typeof require.ensure !== "function") {
-      return false;
-    }
-
     // Make iframe-resizer avaliable to the embed
     // We only care about contentWindow so require that minified file
 
-    require.ensure([], require => {
-      require("iframe-resizer/js/iframeResizer.contentWindow.js");
-    });
+    import("iframe-resizer/js/iframeResizer.contentWindow.js");
```

Comment:
I couldn't test this line due to [iFrameResizer doesn't work on master (v51)](https://github.com/metabase/metabase/issues/47061#top).

Reply:
@uladzimirdev I haven't done with the fix yet but this particular line fixes the issue where `require.ensure` doesn't actually import `iframe-resizer/js/iframeResizer.contentWindow.js` which is a part of the problem of the mentioned issue.

That being said, I approve this line from embedding's point of view.

Reply:
fixed separately